src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -4,12 +4,40 @@
 
 public class StorageImpl<K, V> implements Storage<K, V> {
 
+    public static final int LENGTH = 10;
+    private K[] keyArray;
+    private V[] valueArray;
+    private int count = 0;
```

Comment:
@EliosPride 'count' is incorrect name, better rename to 'iterator' or 'cursor' or 'position', up to you

Reply:
@al-pavlenko rename "count"